src/main/java/domain/Car.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import java.util.Random;
+
+public class Car {
+    private String name;
+    private int movingCnt;
+    private int rnd;
+    private boolean win;
+
+    public Car(String name) {
+        this.name = name;
+        this.movingCnt = 0;
+    }
+
+    public int Moving() {
```

Comment:
```suggestion
    public int moving() {
```
메서드명은 `lowerCamelCase` 명명법을 따라주시면 좋아요.

Reply:
`lowercamelcase` 에 대해서 찾아보니 `스네이크 표기법`도 존재하던데 `lowercamelcase` 를 더 선호하는 이유가 따로 있으신지 궁금합니다!